libcloudforensics/providers/gcp/internal/compute.py

```diff
@@ -569,10 +718,15 @@ def __SshConnection(self) -> None:
     """Create an SSH connection to the virtual machine."""
 
     devnull = open(os.devnull, 'w')
-    subprocess.check_call([
-        'gcloud', 'compute', '--project', self.project_id, 'ssh',
-        '--zone', self.zone, self.name
-    ], stderr=devnull)
+    cmd_list = ['gcloud',
+                'compute',
+                '--project',
+                self.project_id,
+                'ssh',
+                '--zone',
+                self.zone,
+                self.name]
+    subprocess.check_call(cmd_list, stderr=devnull)
```

Comment:
Does this really need its own function? Can't you just call it from `Ssh`?

Reply:
this doesn't belong to this pull request at all it was there from before. I just moved the cmds to a list as per your previous request, we can maybe take this to a another issue? I believe in general SSHing is not the best solution for on instance execution and startup script is more stable way to do that

Reply:
Yes agreed.